### ISSUE TITLE
Truncated download and image scramble

### ISSUE THREAD

**Issue by yimeji** [diomirox/mdownload#2]
Hi, I tried the extension on chrome (still figuring out to load external extension on Edge). I encounter several issues.

(1) When clicked on extension and "get Image" it only downloaded 3 images i.e incomplete download.

(2) The images it download are scrambled

(3) When install this extension "di hola" button appear on bottom left of browser. When click, screen got scroll but nothing happen otherwise.
[<a href="https://imgbb.com/"><img src="https://i.ibb.co/ZMdq2xX/Screenshot-2024-02-08-204224.png" alt="Screenshot-2024-02-08-204224" border="0"></a>](url)

I tried the endtension on the link in the tutorial.

**Comment by Arekkusu1998** [comment]
First of all, thank you very much to @diomirox and @cybinon for Your work!
As regards the extension, I tested it on both Google Chrome and Microsoft Edge. I can confirm all the issues written by @yimeji, with the exception of the last one.

**Comment by diomirox** [comment]
Thank you very much. I will check immediately soon as possible😃

**Comment by diomirox** [comment]
Hi, I tried to fix this issue. However, I've added some extra steps. Please check them out and review the updates. Thank you!

**Comment by yimeji** [comment]
Hi, 
Thank you very much, I tried and it works great!

I do have request and suggestion for the downloader

(1) Option to download the original image format or convert to lossless format.

This is rather personal preference as I rather keep the file in original format (in the site case is webp) or png. Not a fan of jpeg as it has horribal artifact that ruin the image sharpness.

(2) Auto rename of "upload" folder to <"series name"><"chapter no.">.

I tried downloaded 2 different series without renaming the folder and found out that, new files downloaded will replace existing files.
